sources/platform/integrations/ai/qdrant.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Qdrant integration
+description: Learn how to integrate Apify with Qdrant to feed data crawled from the web into the Qdrant vector database.
+sidebar_label: Qdrant
+sidebar_position: 4
+slug: /integrations/qdrant
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+**Learn how to integrate Apify with Qdrant to transfer crawled into the Qdrant vector database.**
+
+---
+
+[Qdrant](https://qdrant.io) is a high performance managed vector database that allows users to store and query dense vectors for next generation AI applications such as recommendation systems, semantic search, and retrieval augmented generation (RAG).
```

Comment:
did you mean qdrant.tech? .io throws error in lychee and I cannot find them at this URL

Reply:
> did you mean qdrant.tech? .io throws error in lychee and I cannot find them at this URL

Thanks @TC-MO  . My bad, I run replace pinecone -> qdrant and I also replaced URLs 🤦🏻